config.yaml

```diff
@@ -67,7 +67,7 @@ params:
       social:
         github: ikanor
         twitter: minifranky
-    mvader:
+    erizocosmico:
```

Comment:
Not all of them ;) : https://github.com/src-d/blog/blob/9c13539ca85770f12985f895e15aa19e83d4c7cc/content/post/proteus.md

Rebase and change this too.